iosHackthonApp/iosHackthonApp/Location/LocationManager.swift

```diff
@@ -12,38 +12,44 @@ import CoreLocation
 
 class LocationManager: NSObject, ObservableObject {
 
-	@Published var userLatitude: Double = 0
-	@Published var userLongitude: Double = 0
-	@Published var address: String = ""
-	
-	private let locationManager = CLLocationManager()
-
-	override init() {
-		super.init()
-		self.locationManager.delegate = self
-		self.locationManager.desiredAccuracy = kCLLocationAccuracyBest
-		self.locationManager.requestWhenInUseAuthorization()
-		self.locationManager.startUpdatingLocation()
-	}
+    @Published var userLatitude: Double = 0
+    @Published var userLongitude: Double = 0
+    @Published var address: String = ""
```

Comment:
aren't the lat & long `Float`?

Reply:
Double is a double sized float :-)

Reply:
Interesting @roger-tan. These are just used locally for us to figure out how far away a meal is from the users current location :-)